api/comments.rb

```diff
@@ -1,3 +1,31 @@
+get "#{APIPREFIX}/comments" do
+  query = Comment.all
+
+  # handle filters
+  error 400, "user_id is required" unless params['user_id']
+  query = query.where(author_id: params['user_id'])
```

Comment:
@felipetrz One thing I completely missed, filter out any comments that have `anonymous` or `anonymous_to_peers` set to true.